draft-voit-rats-attestation-results.md

```diff
@@ -387,11 +398,12 @@ Jump to step (6.1).
      3. Disallow any information exchange into a Relying Party context for which that Verifier B appraised Trustworthiness Vector is not qualified.
 
 As link layer protocols re-authenticate, steps (1) to (2) and steps (3) to (6) will independently refresh.
-In general, there are two types of triggers that trigger a refresh of Evidence generation (1), Attestation Result generation (2), and in consequence AR-augmented Evidence generation (4):
-1. live-cycle events, e.g. a change to an Authentication Secret of the Attester or an update of a software component
-2. uptime-cycle events, e.g. a hard reset of a composite device or a re-initialization of a TEE.
-3. authentication-cycle events, e.g. a link-layer interface resets or new TLS session is spawned.
-Appropriate re-generation of fresh Evidence, fresh Attestation Results, and fresh AR-augmented Evidence allows the trustworthiness of an Attester to be continuously re-appraised.
+This allows the Trustworthiness of Attester to be continuously re-appraised.
+There are only specific triggers which will refresh Evidence generation (1), Attestation Result generation (2), and in consequence AR-augmented Evidence generation (4):
+
+* live-cycle events, e.g. a change to an Authentication Secret of the Attester or an update of a software component
```

Comment:
non-native question: wouldn't this be life-cycle instead?

Reply:
Yes